package.json

```diff
@@ -9,7 +9,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "airtable": "^0.9.0", 
+    "airtable": "^0.9.0",
```

Comment:
I have changed this to previously used version.
Do not upgrade to 0.10.0, as it is killing the application communication with airtable servers.
@sandy9808

Reply:
Ok, i will keep it in mind. 😄